packages/cache/src/index.ts

```diff
@@ -16,8 +21,8 @@ function cacheKey(key: string): string {
   return CACHE_KEY_PREFIX + key;
 }
 
-export async function init() {
-  cacheType = config.questionRenderCacheType;
+export async function cacheInit(config: CacheConfig) {
```

Comment:
Bikeshedding, I know, but while reviewing, I don't necessarily love this naming convention. Should I update function names to `"verb"Cache"` (i.e. `initCache` here instead of `cacheInit`)?

Reply:
Sounds great! I appreciate the added flexibility. I have updated this to use `cache` as a class in [85bf7c4](https://github.com/PrairieLearn/PrairieLearn/pull/9360/commits/85bf7c47890d4ff51f012fcf7fe6f09ea45c002c).